velox/expression/VectorFunction.h

```diff
@@ -152,6 +152,20 @@ class AlwaysFailingVectorFunction final : public VectorFunction {
   std::exception_ptr exceptionPtr_;
 };
 
+// This functions is used when we know a function will never be called because
+// it is default null with a literal null input value. For example like(c0,
+// null).
+class AlwaysNullDefaultNull final : public VectorFunction {
```

Comment:
This name is a bit confusing to me. Wondering if we could renaming to something easier to understand. Maybe, NeverCalled? CC: @bikramSingh91

Reply:
NeverCalledDefaultNull?